src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -10,26 +10,34 @@
 public class Shooter extends LoggedSubsystem<Data, ShooterMap> {
 
     public enum Speeds {
-        FULL_SPEED(4500),
+        FULL_SPEED(3000, 5500, 4500),
 
-        SUBWOOFER_SHOT(3000),
+        SUBWOOFER_SHOT(3000, 3000, 3000),
 
-        PODIUM_SHOT(3500),
+        PODIUM_SHOT(3500, 3500, 3500),
 
-        HALF_SPEED(2250),
+        HALF_SPEED(2250, 2250, 2250),
 
-        AMP_SPEED(1125),
+        AMP_SPEED(1125, 1125, 1125),
 
-        OFF(0);
+        OFF(0, 0, 0);
 
-        private double speed;
+        private double leftSpeed;
+        private double rightSpeed;
```

Comment:
change left and right speed to top and bottom speed

Reply:
I think we want to change these to A and B and then label Alpha and Beta accordingly.